### ISSUE TITLE
Select image uploads a corrupt file iOS 11

### ISSUE THREAD

**Issue by no1itguy** [jamesmontemagno/MediaPlugin#404]
Issue is that if I select an image and post it to an API to save to the server from an iOS device with iOS 11, the file is corrupt and cannot be opened, if I take an image and post to API everything is fine. I do not have the issue on an older device with iOS 9.3.5 and Simulator with iOS 10.1 with the same app version.

**Failure to fill out this information will result in this issue being closed.** If you post a full stack trace in a bug it will be closed, please post it to http://gist.github.com and then post the link here.

## Bug Information

Version Number of Plugin: 3.0.1
Device Tested On: iPod Touch iOS 11.1.2, iPod Touch iOS 9.3.5, iPhone 6S iOS 11.1.2
Simulator Tested On: iPhone 5 iOS
Version of VS: Visual Studio 2017 Enterprise
Version of Xamarin: Xamarin Forms 2.4.0.18342
Versions of other things you are using:

### Steps to reproduce the Behavior
Select image from library and upload to an API

### Expected Behavior
Be able to view the image once in the final destination folder

### Actual Behavior
when moved to final destination error: We can't open this file.

### Code snippet
        public ICommand SelectPictureCommand { get { return new Command(PickPhoto); } }

        private async void PickPhoto()
        {
            await CrossMedia.Current.Initialize();
            MediaFile pic = new MediaFile("", null);
            if (!CrossMedia.Current.IsPickPhotoSupported)
            {
                return;
            }
            var filename = DateTime.Now.ToString("yyyyMMddhhmmss") + ".jpg";
            try
            {
                pic = await CrossMedia.Current.PickPhotoAsync(new PickMediaOptions
                {
                    CompressionQuality = 100,
                    PhotoSize = PhotoSize.MaxWidthHeight
                });
            }
            catch(Exception ex)
            {
                System.Diagnostics.Debug.WriteLine(ex.Message);
                return;
            }
            if (pic == null)
                return;
            Stream imgStream = new MemoryStream();
            pic.GetStream().CopyTo(imgStream);
            ImgStreams.Add(new NCRPics { FileName = filename, Image = pic });
            ImgList.Add(pic.Path);
            SelImage = pic.Path;
        }
### Screenshots
![erroruploaded](https://user-images.githubusercontent.com/31656051/33087067-83a88eee-ceb7-11e7-8db4-d24f145dd6ad.JPG)

**Comment by jamesmontemagno** [comment]
Does it display locally on iOS?

**Comment by no1itguy** [comment]
I am not sure why you closed this issue, I have found a consistent issue, and after several test, when I take a picture with TakePhoto the file is uploaded without an issue, but when I select the image from the library it ends up corrupted, this is from the same device and the same image, it can only be the the plugin, I am missing something.
But I did notice when I take a picture the file size is about 500kb but when I select the same image it is about 1700kb. I have the compression quality and file size set to the same on both.
